coalib/settings/Section.py

```diff
@@ -41,7 +42,8 @@ def append_to_sections(sections,
         sections[section_name.lower()] = Section(section_name)
 
     sections[section_name.lower()].append(Setting(
-        key, str(value), origin, from_cli=from_cli, to_append=to_append))
+        key, str(value), SourcePosition(origin),
```

Comment:
I'm only 90%-ish sure about this ... so feel free to tell me why I am wrong here...

this shouldnt be necessary, again because it doesnt add a line number.

If tests are breaking because of this, my guess is that means something isnt working properly when Setting.origin is not a `SourcePosition`

Reply:
This does seem to be unnecessary. I will remove it.